homeassistant/components/media_player/webostv.py

```diff
@@ -158,15 +171,14 @@ def __init__(self, host, name, customize):
         self._volume = 0
         self._current_source = None
         self._current_source_id = None
-        self._source_list = None
         self._state = STATE_UNKNOWN
-        self._app_list = None
-
-        self.update()
```

Comment:
I decided to pass the parameter to the callback instead. Sorry for the confusion.